Quadriga/src/main/webapp/WEB-INF/views/settings/addpublicpage.jsp

```diff
@@ -6,24 +6,30 @@
 	href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
 <script>
 //@ sourceURL=filename.js
+	
+	$(document).ready(function(){
+	$("#mainPageTab").attr('class','list-group-item');
+	$("#aboutTab").attr('class','list-group-item disabled');
+    });
```

Comment:
Use tiles attributes instead of javascript. take a look at how the main navigation is set active and do it similarly.